mcs/build/tests.make

```diff
@@ -96,8 +96,10 @@ endif
 
 test_assemblies :=
 
+test_lib_output = $(topdir)/class/lib/$(PROFILE)/$(test_lib)
```

Comment:
If the tests are not currently relocatable, this PR is going to need to change that as well if we hope to run them statically linked